### ISSUE TITLE
Is there a maintained fork of ozzo-validation?

### ISSUE THREAD

**Issue by lazharichir** [go-ozzo/ozzo-validation#162]
Seems like this library is no longer maintained. Is there an actively maintained fork out there?

Thank you!

**Comment by gradinarot** [comment]
Would like to know too

**Comment by fr3fou** [comment]
I'd also like to know! This is the only validation library that I've actually enjoyed using

**Comment by ysomad** [comment]
Also would like too know!

**Comment by ltns35** [comment]
I've made a fork of this project and I'm planning to maintain it, any help it's welcome. For the moment I only upgrade the Go version to v1.18

https://github.com/ltns35/ozzo-validation

**Comment by swithek** [comment]
We have an actively maintained [fork](https://github.com/jellydator/validation) as well. It has a few additions and fixes, most of which are from the PRs found in this repository.

**Comment by ysomad** [comment]
> We have an actively maintained [fork](https://github.com/jellydator/validation) as well. It has a few additions and fixes, most of which are from the PRs found in this repository.

How is perfomance compared to https://github.com/go-playground/validator ?

**Comment by swithek** [comment]
@ysomad The core functionality of that fork is the same as ozzo-validation's, which means the performance is also the same. I haven't compared this with https://github.com/go-playground/validator but I also don't see any reason to do that either as both libraries are using the reflection package under the hood.

**Comment by desprit** [comment]
I'm using https://github.com/gookit/validate which is also good. Not sure about performance though.

**Comment by kolkov** [comment]
Yes — the library is actively maintained again. I've taken over as maintainer with full org access. See #207 for the full plan: CI modernization, merging the backlog of PRs, fixing known bugs, and cutting a new release.

Thanks to everyone who kept the community going in the meantime, and a shout-out to the jellydator/validation fork for keeping things alive.
